collections/nemo_tts/nemo_tts/data_layers.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2019 NVIDIA Corporation
+import torch
+try:
+    from apex import amp
+except AttributeError:
```

Comment:
could you please change this to: "except (AttributeError, ModuleNotFoundError) as e:"

Reply:
amp is no longer used in this file and will be removed

Reply:
can you please remove this try except block then?